background.js

```diff
@@ -0,0 +1,29 @@
+class BackGround {
+  static HIT_NON = 0;
+  static HIT_LEFT = 1;
+  static HIT_RIGHT = 2;
+  static HIT_TOP = 3;
+  static HIT_BOTTOM = 4;
+  constructor() {}
+  Draw(context) {
```

Comment:
一般的に関数はローワーキャメルケースで命名します。
他の箇所についても同様です。

```suggestion
  draw(context) {
```

Reply:
承知しました。
全て修正するのは時間を割いてしまうので、次回から気をつけます。

Reply:
はい、一旦ここでは全部はやらなくていいです